packages/nextjs/package.json

```diff
@@ -13,6 +13,16 @@
   "module": "build/esm/index.server.js",
   "browser": "build/esm/index.client.js",
   "types": "build/types/index.types.d.ts",
+  "exports": {
```

Comment:
Curious if this will cause any problems but given that tests pass, let's see 🤔

Reply:
we can never know... 😶‍🌫️

Reply:
We can check with https://arethetypeswrong.github.io/